cli/src/pcluster/aws/ec2.py

```diff
@@ -361,4 +363,8 @@ def describe_volume(self, volume_id):
     @AWSExceptionHandler.handle_client_exception
     def run_instances(self, **kwargs):
         """Describe network interfaces."""
-        return self._client.run_instances(**kwargs)
+        try:
+            self._client.run_instances(**kwargs)
+        except ClientError as e:
+            if e.response.get("Error").get("Code") != "DryRunOperation":
```

Comment:
It is ok to not propagate the dry run exception to the console, but what about keeping track of it with a warning within the log file?

Reply:
Our original behavior ignored DryRun exception right away. So I would say it is fine to be consistent with our original behavior